types/seat/wlr_seat_keyboard.c

```diff
@@ -338,8 +345,25 @@ static void seat_client_send_keymap(struct wlr_seat_client *client,
 			continue;
 		}
 
+		keymap_fd = os_create_anonymous_file(keyboard->keymap_size);
```

Comment:
This is never closed on the compositor side. It should be closed right after `wl_keyboard_send_keymap`